helm/coredns-app/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 appVersion: 1.9.3
 description: A Helm chart for CoreDNS
+icon: https://s.giantswarm.io/app-icons/coredns/1/dark.svg
```

Comment:
When I open this link in a browser it loads an svg, but it's all white. This is probably not intended?

Reply:
Probably because it's a white logo in white background. Converted it to PNG, looks as follows:
<img width="362" alt="logo" src="https://user-images.githubusercontent.com/935479/196407090-017ba41e-d40a-40b5-bd78-b2029c81d2b8.png">

Reply:
That is intended. `dark.svg` means that it is an icon we use in dark mode, which at the moment is the default for happa.